Shifty/BrowserManager.swift

```diff
@@ -242,7 +242,7 @@ enum BrowserManager {
             logw("Error: Could not get url, app already closed")
             return nil
         }
-        guard let window = (browser.windows?() as? [Window])?.first else {
+        guard let windows = browser.windows?(), windows.count > 0, let window = windows[0] as? Window else {
```

Comment:
`let window = windows.firstObject` would remove the need to check `windows.count`. It would still do one internally, but it would make the code a bit nicer :)

Reply:
Thanks and [done](https://github.com/thompsonate/Shifty/commit/b57b5932cbee4f4306e07b0367fa17c2737c005f) :)